### PR TITLE
Update dedup/merge docs with output data stream

### LINKED ISSUE #792: Update documentation about Duplicate detection to explain output data streams

### PR DESCRIPTION
**Pull request by LosD**
This updates the dedup and merge docs so output data streams are used as the default.

This also adds CSS support for the DocBook  ```<warning>``` element, as well as update styling a bit for ```<tip>``` and ```<warning>``` and ```<mediaobject>```, giving them a bit of air.

Fixes #792

### CODE REVIEW

**Comment by LosD** [comment]
New warning box:
image

**Comment by LosD** [comment]
Unfortunately, the configuration windows aren't exactly pretty with the overlap between name and scope button:
image

**Comment by LosD** [comment]
Title-on-top-of-button fixed. I kinda shoehorned the overlap fix of the actual DataCleaner into this PR.

**Comment by ClaudiaPHI** [comment]
:+1:

File: documentation/src/docbkx/chapter22-improve.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <chapter xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	id="improve" xmlns="http://www.oasis-open.org/docbook/4.5" xmlns:xl="http://www.w3.org/1999/xlink"
```
Comment (kaspersorensen) [inline documentation/src/docbkx/chapter22-improve.xml]: Why did you delete "improve" id here?

**Comment by kaspersorensen** [comment]
LGTM except for the missing id which I didn't understand :)

File: documentation/src/docbkx/chapter22-improve.xml (hunk 1 above)
Comment (LosD) [inline documentation/src/docbkx/chapter22-improve.xml]: Err... Eeeehhhmm... Yeah. I misunderstood the purpose of "id" in chapter... I actually thought I put it back, but obviously not.

**Comment by LosD** [comment]
It's back :)
